cli/tests/test_status_command.rs

```diff
@@ -368,3 +368,67 @@ fn test_status_simplify_conflict_sides() {
     Then run `jj squash` to move the resolution into the conflicted commit.
     "###);
 }
+
+#[test]
+fn test_status_untracked_files() {
+    let test_env = TestEnvironment::default();
+    test_env.add_config(r#"snapshot.auto-track = "none()""#);
+
+    test_env.jj_cmd_ok(test_env.env_root(), &["git", "init", "repo"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("initially-untracked-file"), "...").unwrap();
+    std::fs::write(repo_path.join("always-untracked-file"), "...").unwrap();
+
+    let stdout = test_env.jj_cmd_success(&repo_path, &["status"]);
+    insta::assert_snapshot!(stdout, @r"
+    The working copy is clean
+    ? always-untracked-file
+    ? initially-untracked-file
+    Working copy : qpvuntsm 230dd059 (empty) (no description set)
+    Parent commit: zzzzzzzz 00000000 (empty) (no description set)
+    ");
+
+    test_env.jj_cmd_success(&repo_path, &["file", "track", "initially-untracked-file"]);
+
+    let stdout = test_env.jj_cmd_success(&repo_path, &["status"]);
+    insta::assert_snapshot!(stdout, @r"
+    Working copy changes:
+    A initially-untracked-file
+    ? always-untracked-file
```

Comment:
nit: might be better to insert `Untracked paths:` header. These two lists are sorted separately.